Dockerfile

```diff
@@ -1,8 +1,15 @@
-FROM node:10.17-alpine
+FROM docker.io/library/node:14-alpine
 
-WORKDIR /app
-COPY . /app
 ENV TZ=Europe/Berlin
 EXPOSE 3033
-RUN chown -R 1000:1000 /app
+
+RUN apk add --no-cache autoconf automake build-base make nasm zlib-dev
+#RUN npm install -g gulp
+
+WORKDIR /app
+COPY package.json package-lock.json ./
+RUN npm ci && npm cache clean --force
+COPY . .
+RUN node node_modules/gulp/bin/gulp.js
```

Comment:
ok, I was not sure we want to run everything on startup, but we have similar on client, SHD pods have little use, etc

Reply:
gulp run very fast for shd, but can we use npx gulp for it? @Loki-Afro